docs/guides/migration_guide_v2.md

```diff
@@ -66,11 +66,57 @@ Below you find an overview of the changes in environment variables:
 | `SCW_REGION`            | `SCW_DEFAULT_REGION`                        |
 | `SCW_TOKEN`             | `SCW_SECRET_KEY`                            |
 
-~> **Important:** `SCALEWAY_ACCESS_KEY` was changed to `SCW_ACCESS_KEY`. This should be your access key (e.g. `SCWZFD9BPQ4TZ14SM1YS`). Your secret key (previously known as _token_) must be set in `SCW_SECRET_KEY` (`xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`).
+~> **Important:** `SCALEWAY_ACCESS_KEY` was changed to `SCW_ACCESS_KEY`.
+This should be your access key (e.g. `SCWZFD9BPQ4TZ14SM1YS`).
+Your secret key (previously known as _token_) must be set in `SCW_SECRET_KEY` (`xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`).
+
+Terraform can also read standard [Scaleway configuration files](https://github.com/scaleway/scaleway-sdk-go/blob/master/scw/README.md).
+By doing so, you can use the same configuration between different tools such as the [CLI](https://github.com/scaleway/scaleway-cli) or [Packer](https://www.packer.io/docs/builders/scaleway).
 
 ## Resources
 
-All resources are from now on prefixed by `scaleway`, their product category and their product name (`scaleway_{product-category-name}_{product-name}_{resource-name}`). For instances an S3 bucket belongs to the `Storage` product category and is a resource of the `Object` product. Hence it is named: `scaleway_object_bucket`.
+All resources are from now on prefixed by `scaleway`, their product category and their product name (`scaleway_{product-category-name}_{product-name}_{resource-name}`).
+For instances an S3 bucket belongs to the `Storage` product category and is a resource of the `Object` product.
+Hence it is named: `scaleway_object_bucket`.
+
+### Migration guide for renamed resources
+
+Because the resources changed their name, we cannot use automatic state migration.
+We will first manually remove the resource from the terraform state and then use [`terraform import`](https://www.terraform.io/docs/import/usage.html) to import existing resources to a renamed resource.
+
+For instance, let's suppose that you have resource in `fr-par-1` such as:
+
+```hcl-terraform
+provider "scaleway" {
+    zone= "fr-par-1"
+}
+
+resource scaleway_server main {
+  name  = "foobar"
+  type  = "DEV1-S"
+  image = "cf44b8f5-77e2-42ed-8f1e-09ed5bb028fc"
+}
+```
+
+First, let's delete the resource from your terraform state using the [`terraform state`](https://www.terraform.io/docs/commands/state/index.html) command.
```

Comment:
Before RM maybe add a command to get the ID in an env variable that can be used for import